woocommerce/compatibility/class-sv-wc-order-compatibility.php

```diff
@@ -522,6 +524,64 @@ public static function get_edit_order_url( \WC_Order $order ) {
 	}
 
 
+	/**
+	 * Returns the admin screen id for orders.
+	 *
+	 * This method detects the screen_id according to HPOS availability,
+	 * as cannot rely anymore on the shop_order registered post type as the screen ID.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return string
+	 */
+	public static function get_screen_id( ) : string {
+
+		return SV_WC_Plugin_Compatibility::is_hpos_enabled() ? wc_get_page_screen_id( 'shop-order' ) : 'shop_order';
+	}
+
+
+	/**
+	 * Determines whether the given $post_id is a WooCommerce order or not according to HPOS availability.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param int $post_id post ID.
+	 * @return bool
+	 */
+	public static function is_order( int $post_id ) : bool {
```

Comment:
consider renaming method to is_order_admin_screen() : bool to avoid confusion with determining if a a post ID is an order; additionally, you could make this method to work also without passing a $post_id (make the post_id arg nullable) by just checking if the current page is an order admin screen

Reply:
my bad I misinterpreted this